pallets/xst/src/lib.rs

```diff
@@ -183,23 +178,105 @@ pub mod pallet {
             Ok(().into())
         }
 
-        #[pallet::call_index(2)]
+        #[pallet::call_index(1)]
         #[pallet::weight(<T as Config>::WeightInfo::enable_synthetic_asset())]
         pub fn enable_synthetic_asset(
+            origin: OriginFor<T>,
+            asset_id: T::AssetId,
+            reference_symbol: T::Symbol,
+            fee_ratio: Fixed,
+        ) -> DispatchResultWithPostInfo {
+            ensure_root(origin)?;
+
+            Self::enable_synthetic_asset_unchecked(asset_id, reference_symbol, fee_ratio, true)?;
+            Ok(().into())
+        }
+
+        #[pallet::call_index(3)]
+        #[pallet::weight(<T as Config>::WeightInfo::register_synthetic_asset())]
+        pub fn register_synthetic_asset(
+            origin: OriginFor<T>,
+            asset_symbol: AssetSymbol,
+            asset_name: AssetName,
+            reference_symbol: T::Symbol,
+        ) -> DispatchResultWithPostInfo {
+            ensure_root(origin)?;
+
+            let synthetic_asset_id: T::AssetId =
+                AssetId32::<common::PredefinedAssetId>::from_synthetic_reference_symbol(
+                    &reference_symbol,
+                )
+                .into();
+
+            Self::register_synthetic_asset_unchecked(synthetic_asset_id, asset_symbol, asset_name)?;
+
```

Comment:
I think we need to enable registered asset or it will require 2 referendums to add new synthetic asset